bin/NativeTests/JsRTApiTest.cpp

```diff
@@ -1478,17 +1478,18 @@ namespace JsRTApiTest
     {
         JsValueRef result = JS_INVALID_REFERENCE, callbackState = JS_INVALID_REFERENCE, exception = JS_INVALID_REFERENCE;
         JsValueType cbStateType = JsUndefined;
+        const wchar_t *script = _u("new Promise((res, rej) => res()).then(() => 1)");
```

Comment:
if I convert to WCHAR, should I also convert to JsRun? Or, if making these tests xplat is a goal, we should probably just open an issue to do the conversion all at once.